zstackctl/zstackctl/ctl.py

```
@@ -5036,6 +5036,17 @@ def run(self, args):
         if args.host3_info is not False:
             copy(copy_arg, self.host2_post_info)
 
+        #sync libvirt TLS CA key
+        libvirt_ca_dir = "/var/lib/zstack/pki/CA"
+        if os.path.isfile("%s/cacert.pem" % libvirt_ca_dir) and os.path.isfile("%s/cakey.pem" % libvirt_ca_dir):
```

Ruff (0.15.5)

[error] `os` may be undefined, or defined from star imports

(F405)